tests/shutdown/cleanup_before_shutdown.pm

```diff
@@ -50,16 +50,16 @@ END_SCRIPT
 
     # Proceed with dhcp cleanup on qemu backend only.
     # Cleanup is made, because if same hdd image used in multimachine scenario
-    # on several nodes, the dhcp clients use same id and cause conflicts on dhcpd server.
-    if (is_qemu) {
+    # on several nodes, the dhcp clients use same DUID and cause conflicts on dhcpd server.
+    if (is_qemu || is_svirt) {
```

Comment:
If we don't need this change on s390x, how about `is_svirt_except_s390x`

Reply:
Does it make any difference on s390? Can you give me example test run I can verify on?

Reply:
for 390x, we use static ip address  defined in https://gitlab.suse.de/openqa/salt-pillars-openqa/-/blob/master/openqa/workerconf.sls

Reply:
Anyway, I don't think use `is_svirt` can lead to any issue, we can use this workaround for s390x as well.

Reply:
That's why we do `rm -f` so if those files are missing we will just proceed. Can you give me example test run I can verify on?

Reply:
https://openqa.suse.de/tests/9678944#dependencies